src/js/components/__tests__/ServiceDetailConfigurationTab-test.js

```diff
@@ -42,10 +42,10 @@ describe('ServiceDetailConfigurationTab', function () {
   describe('#render', function () {
 
     it('renders correct id', function () {
-      var configurationView = TestUtils
-        .findRenderedComponentWithType(this.instance, ConfigurationView);
+      var ServiceSpecView = TestUtils
```

Comment:
#nit Variable names (ex. const)  should start with a lower case letter e.g. `serviceSpecView`